explore/src/main/scala/explore/TopBar.scala

```diff
@@ -140,11 +144,23 @@ object TopBar {
                                    icon = Icons.Logout.fixedWidth(),
                                    onClick = logout.runAsync
                       ),
+                      DropdownItem()(
+                        Icons.BarCodeRead.fixedWidth(),
+                        "Log Level",
+                        DropdownMenu(
+                          DropdownItem(onClick = setLogLevel(LogLevelDesc.INFO))(
+                            Checkbox(label = "Info", checked = level != LogLevelDesc.DEBUG)
+                          ),
+                          DropdownItem(onClick = setLogLevel(LogLevelDesc.DEBUG))(
+                            Checkbox(label = "Debug", checked = level == LogLevelLogger.Level.DEBUG)
```

Comment:
Should this be `checked level == LogLevelDesc.DEBUG`?

Reply:
Maybe we can use `===`? (Sorry to insist :D but the `Eq` would come for free with `Enumerated` )

Reply:
Good suggestion. I'll add the `Enumerated`